### PR TITLE
Custom slack message

### PR DESCRIPTION
**Pull request by jayco**
This PR relates to https://github.com/buildkite/buildkite/pull/5282, the ability to add a custom slack notification message 😀

### CODE REVIEW

**Comment by eleanorakh** [comment]
@jayco awesome! I say ship this, and  the docs update can come after   :)

**Comment by eleanorakh** [comment]
Turns out clicking `back` didn't work!!! This still needs review, I mean ship the change this docs PR depends on 🙃🙃🙃

File: pages/pipelines/notifications.md.erb [hunk 1]
```diff
@@ -149,6 +149,19 @@ notify:
...
+
+```yaml
+notify:
+  - slack:
+      - "buildkite-community#sre"
+      message: "SRE related information here..."
+  - slack:
+      - "buildkite-community#announcements"
+      message: "General announcement for the team here..."
+```
```
Comment (sj26) [inline pages/pipelines/notifications.md.erb:162]: I don't think these examples are valid yaml.

Does this work?

```yaml
notify:
   - slack: "buildkite#announcements"
     message: "New release"
   - slack:
       channels:
         - "buildkite#deploy"
         - "buildkite#infrastructure"
       message: "New release of buildkite deployed, ${BUILDKITE_COMMIT} is now in ${ENVIRONMENT}"
```

Comment (jayco) [reply pages/pipelines/notifications.md.erb:164]: 😅 good catch - have updated

**Comment by eleanorakh** [comment]
This looks good for merging!

**Comment by plaindocs** [comment]
Oops, hope this wasn't waiting on me/

**Pull request merged**